src/navsat_transform.cpp

```diff
@@ -271,8 +269,8 @@ void NavSatTransform::computeTransform()
 }
 
 bool NavSatTransform::datumCallback(
-  robot_localization::srv::set_datum::request & request,
-  robot_localization::srv::set_datum::response &)
+  std::shared_ptr<robot_localization::srv::SetDatum::Request> request,
```

Comment:
Potentially stupid question: did the service types not expose a `::SharedPtr` alias?

Reply:
It did :) fixed!